Test/Float/FloatArithmeticOperators.pml

```diff
@@ -0,0 +1,55 @@
+// This is needed because dot notation is not yet supported by the lexer
+hidden float tmpN;
+hidden float tmpD;
+
+inline makeFloat(f, nominator, denominator) {
+    tmpN = nominator;
+    tmpD = denominator;
+    f = tmpN / tmpD;
+}
+
+active proctype test() {
+    float a = 14.25;
+    float b = -a;
+    int c = 14;
+    // float var is properly compared with int
+    assert(c != a);
+    assert(14.0 != 0);
+    assert(a < 15);
+    assert(a > c);
+    assert(a >= c);
+    assert(a <= 15)
+
+    // float var is properly compared to float
+    assert(a != b);
+    assert(b == b);
+    assert(a < 15.6);
```

Comment:
Could we check against 14.3?

Reply:
of course - the reason for 14.0 was the problem with internal casting of float values into int values inside of eval function (run.c) - a result of it was that assert(14.4 == 14) was ok... also assert(14.4 == 14.3) because both values were casted to 14 before comparison... - anyway - I will change it to 14.3 - no problemo

Reply:
done